app/src/main/java/com/eveningoutpost/dexdrip/NFCReaderX.java

```diff
@@ -474,13 +480,27 @@ protected void onPostExecute(Tag tag) {
             }
         }
 
+        void LogLibre2StartStreeming(ENABLE_STREAMING success, String extraData) {
```

Comment:
Spelling typo, I assume this should be streaming not streeming

Reply:
Fixed.